default-client/src/main/java/ru/qa/api/client/DefaultResponse.java

```diff
@@ -0,0 +1,43 @@
+package ru.qa.api.client;
+
+import com.google.gson.Gson;
+import ru.qa.api.client.exceptions.APIAssertionError;
+import ru.qa.api.client.interfaces.ResponseActions;
+
+import java.net.http.HttpResponse;
+
+public class DefaultResponse implements ResponseActions {
+
+    HttpResponse<String> response;
+
+    public DefaultResponse(HttpResponse response) {
```

Comment:
Мб добавить дженерик как в атрибуте?
```suggestion
    public DefaultResponse(HttpResponse<String> response) {
```

Reply:
Я пока думаю, разрулить сие дженериком, разными типами под одним интерфейсом или сделать God Object, который будет работать с любым типом ответов. 

Третий варик нравится меньше всего
Первый - серединка, как будто, полумера
Второй привлекает больше всего, будет общий интерфейс, но подход к обработке ответов будет чуток отличаться в зависимости от типа принимаемого ответа.

Reply:
Ну я отталкиваюсь от реализации. Сейчас, если в тело ответа положить что-то другое, то при вызове реализованных методов будет `ClassCastException`
Мне такие вещи не особо нравятся, поэтому по мне честно заявить, что "пока только `String`"
Лан, мб сильно заморачиваюсь, будем ждать "тела" для "дела" 🤣